rados/rados_test.go

```diff
@@ -407,6 +407,59 @@ func (suite *RadosTestSuite) TestWaitForLatestOSDMap() {
 	assert.NoError(suite.T(), err)
 }
 
+func (suite *RadosTestSuite) TestSetLocator() {
+	suite.SetupConnection()
+
+	// Set Locator Key
+	suite.ioctx.SetLocator("Locator123")
+
+	// Write and Read
+	for i := 1; i <= 50; i++ {
```

Comment:
Why this number of iterations?

Reply:
Because `setLocator` will try to place all objects in the same pg. Therefore I try to create a higher amount of objects, and look if they are really places on the same one. I could use two, but I think its safer to use more to make sure it wasn't a coincidence. Any suggestions ?

Reply:
Ah, I think it then makes even more sense to try and validate the behavior with and without setting the locator (see the other conversation).